jsk_apc2016_common/node_scripts/polygon_array_to_collision_object.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+
+import time
+
+import tf
+from std_msgs.msg import String
+from geometry_msgs.msg import PointStamped
+from geometry_msgs.msg import PoseStamped
+from geometry_msgs.msg import Quaternion
+from geometry_msgs.msg import TransformStamped
+from geometry_msgs.msg import Vector3
+from jsk_recognition_msgs.msg import PolygonArray
+import moveit_commander
+import rospy
+import tf2_geometry_msgs.tf2_geometry_msgs as tf_utils
+
+import numpy as np
+
+flag = 1
+
+
+def callback_flag(msg):
+    global flag
```

Comment:
I wonder if there is a smarter way to write this node...
try using ConnectionBasedTransport with Class